### PR TITLE
ec2_key - clean up examples

### PR DESCRIPTION
**Pull request by prasadkatti**
##### SUMMARY
- cleaned up ec2_key examples
- added an example showing how to provide key_material using file lookup plugin

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
`ec2_key`

##### ANSIBLE VERSION

```
devel
```

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/ec2_key.py [hunk 1]
```diff
@@ -62,44 +62,39 @@
...
-    key_material: 'ssh-rsa AAAAxyz...== me@example.com'
-    force: false
-    state: present
+    name: my_keypair
+    key_material: "{{ lookup('file', '/path/to/public_key/id_rsa.pub') }}"
 
-# Creates a new ec2 key pair named `example` if not present using provided key
-# material
-- name: example3 ec2 key
+- name: another way to create a key pair using key_material from a file
```
Comment (willthames) [inline lib/ansible/modules/cloud/amazon/ec2_key.py]: Not convinced we need this example, particularly with `with_` going away.

**Review by willthames — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/amazon/ec2_key.py (hunk 1 above)
Comment (prasadkatti) [reply lib/ansible/modules/cloud/amazon/ec2_key.py]: Sorry, I didn't know about `with_` going away. I have replaced the already existing `with_file` example with a `lookup('file', 'id_rsa.pub')` example.

**Comment by prasadkatti** [comment]
ready_for_review

**Review by ryansb — APPROVED** [review]

**Review by dharmabumstead — APPROVED** [review]

**Review by willthames — APPROVED** [review]

**Comment by willthames** [comment]
Merged, thanks @prasadkatti

**Pull request merged**